src/calendar/src/styles/index.cssr.ts

```diff
@@ -120,6 +120,7 @@ export default c([
           `)
         ])
       ]),
+      cM('no-drop', 'cursor: no-drop;'),
```

Comment:
<img width="832" alt="image" src="https://user-images.githubusercontent.com/18677354/162054001-dcccb470-f22f-45fc-8809-48aaea8c254d.png">

use not-allowed

In semantic, it should be not-allowed